spec/models/use_cases/localize_city_code_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe UseCases::LocalizeCityCode do
+  describe 'in_maubeuge?' do
+    let(:in_maubeuge) { described_class.new(city_code).in_maubeuge? }
+
+    context 'is not in maubeuge perimeter' do
+      let(:city_code) { 75_108 }
+
+      it 'returns false' do
+        expect(in_maubeuge).to be_falsey
```

Comment:
Change `be_falsey` for `be false` because we don't want this test to pass if value is `nil`.